test/release/examples/primers/chplvis/chplvis.html

```diff
@@ -0,0 +1,312 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+</head>
+<body bgcolor="ffffff">
+<div>
+<img src="chapel-a.png" style="float:left">
+<img src="cray-a.png" style="float:right">
+<div style="text-align:center">
+<pre>
+
+
+
+</pre>
+<h1>chplvis</h1>
+<h3>A task and communication debug tool for Chapel</h3>
+</div>
+</div>
+<br/>
+<div style="margin:40px">
+<p>
+<em>chplvis</em> is a tool to help the Chapel program visualize their
+Chapel program's tasks and communication between locales.  Using the
+standard module <em>VisualDebug</em>, the programmer controls what part
+of their program generates information for chplvis.  During the run of
+a program using the <em>VisualDebug</em> module, data files are
+created that are used as input for <em>chplvis</em>.  This document
+will help you understand the <em>VisualDebug</em> module and the
+<em>chplvis</em> tool.
+</p>
+
+<h3>Setup</h3>
+<p>
+<em>chplvis</em> is built by giving the command <em>"make chplvis"</em>
+at the top level of the chapel tree.  This also builds
+the GUI tool, <em>FLTK</em>, required to build and
+run <em>chplvis</em>.  (Note: Some versions of Linux may require the
+standard package <em>libx11-dev</em> to be installed before <em>FLTK</em> will
+compile properly.)  To get the most out of this primer, you should
+compile and run the example programs and examine
+the <em>VisualDebug</em> results with <em>chplvis</em>.  The graphics
+in this primer were produced on a system using the <em>fifo</em> package
+instead of <em>qthreads</em> for the tasking layer.   If you use
```

Comment:
I would suggest changing "instead of qthreads" to "instead of the default, qthreads,"